airflow/utils/dag_processing.py

```diff
@@ -1394,3 +1196,8 @@ def emit_metrics(self):
         # TODO: Remove before Airflow 2.0
         Stats.gauge('collect_dags', parse_time)
         Stats.gauge('dagbag_import_errors', sum(stat.import_errors for stat in self._file_stats.values()))
+
+    # pylint: disable=missing-docstring
+    @property
+    def file_paths(self):
+        return self._file_paths
```

Comment:
API change?

Reply:
We just add a new property. This is an additional property added, without breaking current API. Why do you think it's an API change ?